plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerClient.java

```diff
@@ -137,7 +137,7 @@ public void testImplementSum()
                 Optional.empty()); // filter not supported
     }
 
-    private void testImplementAggregation(AggregateFunction aggregateFunction, Map<String, ColumnHandle> assignments, Optional<String> expectedExpression)
+    private static void testImplementAggregation(AggregateFunction aggregateFunction, Map<String, ColumnHandle> assignments, Optional<String> expectedExpression)
```

Comment:
Same applies to all `TestXxxClient` classes, as they are copies.
Not much benefit though IMO

Reply:
I got warnings in IDE. I don't like warnings, but I do like static methods.

Reply:
i don't have a warning here

Reply:
See also: https://errorprone.info/bugpattern/MethodCanBeStatic

Reply:
Did we decide to use that?

> I got warnings in IDE. I don't like warnings, but I do like static methods.

i don't like either. 

but "method can be static" is test code is counter productive -- the tests themselves are isntance methods, so we shouldn't care if a test helper is an instance method as well, especially when it implements a test's logic (as case here) as compared to being ordinary utility method (those should definitely be static)